src/js/libcs/RenderBackends.js

```diff
@@ -868,3 +868,9 @@ globalInstance.exportPDF = function() {
         exportWith(PdfWriterContext, wnd);
     });
 };
+
+globalInstance.exportPNG = function() {
+    var data = csctx.canvas.toDataURL();
+    var wnd = window.open('about:blank', '_blank');
+    wnd.location.href = data;
```

Comment:
This is a synchronous operation, so you might as well avoid using the `about:blank` URL first, and instead directly use the URL of the PNG.